src/ui/public/kbn_share/kbn_share.less

```diff
@@ -0,0 +1,42 @@
+.sharing-ui-menu__List {
+  display: block;
+  margin: 0;
+  padding: 0;
+}
+
+.sharing-ui-menu__ListWrapper,
+.sharing-ui-menu__SharePanel {
```

Comment:
I feel like we need a strong grip on what we want to do when it comes to naming classes. As it stands, we're not going for a "lax" approach such as "just define a unique component namespace and do whatever you want in there, as in `.abc-list`, `.abc-list-has-item`, `.abc-item`, etc. Having both a namespace that's not clearly delimited (because you could have say `.shareButton` and `.shareButtonText`, and at the same time having component names is bound to be confusing.